src/components/Profile/components/MembershipsList/index.jsx

```diff
@@ -95,7 +95,7 @@ const MembershipsList = ({ username, myProf, createSnackbar }) => {
 
   const transcriptButton = myProf && (
     <Grid container justifyContent="center">
-      <Link className="gc360_link" to="/transcript">
+      <Link to="/transcript">
```

Comment:
This was removed to get rid of the gray box that appeared due to reusing this style from the right side nav, we may want to migrate the styles used in right side nav links to their own CSS file, instead of global at some point